t/errlog.t

```diff
@@ -1111,3 +1111,123 @@ enter 11
 "
 ]
 --- skip_nginx: 3: <1.11.2
+
+
+
+
+=== TEST 29: ringbuf overflow bug
+--- http_config
+    lua_capture_error_log 4k;
+--- config
+    location /t {
+        access_by_lua_block {
+            local errlog = require "ngx.errlog"
+            local msg = string.rep("*", 10)
+
+            for i = 1, 10 do
```

Comment:
How about we only add a small amount of error log here? we will get all the error logs finally(small amount). 

This is more convenient to check the contents of the log more stringent, like this: https://github.com/openresty/lua-resty-core/blob/master/t/errlog.t#L615